front/src/reducers/editor/index.ts

```diff
@@ -261,57 +265,10 @@ export function save(
 }
 
 export type EditorActions =
-  | ActionLoadDataModel
-  | ActionSave
-  | ActionSelectLayers
-  | ActionUpdateTotalsIssue
-  | ActionUpdateFiltersIssue;
-
-//
-// State definition
-// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-export const initialState: EditorState = {
-  // Definition of entities (json schema)
-  editorSchema: [],
-  // ID of selected layers on which we are working
-  editorLayers: new Set(['track_sections', 'errors']),
-  // Editor issue management
-  issues: {
-    total: 0,
-    filterTotal: 0,
-    filterLevel: 'all',
-    filterType: null,
-  },
-};
-
-//
-// State reducer
-// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-const reducer = (inputState: EditorState | undefined, action: EditorActions) => {
-  const state = inputState || initialState;
-
-  return produce(state, (draft) => {
-    switch (action.type) {
-      case SELECT_LAYERS:
-        draft.editorLayers = action.layers;
-        break;
-      case LOAD_DATA_MODEL:
-        draft.editorSchema = action.schema;
-        break;
-      case UPDATE_FILTERS_ISSUE:
-      case UPDATE_TOTALS_ISSUE:
-        draft.issues = {
-          ...state.issues,
-          ...action.issues,
-        };
-        break;
-      default:
-        // Nothing to do here
-        break;
-    }
-  });
-};
+  | editorSliceActionsType['loadDataModelAction']
+  | editorSliceActionsType['selectLayers']
+  | editorSliceActionsType['updateFiltersIssueAction']
+  | editorSliceActionsType['updateTotalsIssueAction']
+  | ActionSave;
```

Comment:
why doesn’t this one become part of the slice ?

Reply:
What do you mean by "this one" ? I think he re-created all the actions

Reply:
no this one is kept outside of the slice, but I guess it’s because it’s a thunk action

const SAVE = 'editor/SAVE';